corehq/apps/users/templates/users/partial/basic_info_js_inline.html

```diff
@@ -4,10 +4,6 @@
 
 <script type="text/javascript">
     $(function() {
-        $('#id_add_phone_number').keydown(function(event) {
-            $("#phone_number_paste_error").css("display", "none");
-            return allowNumeric(event);
-        });
```

Comment:
This made it so that if you tried to type anything that wasn't a number, it just wouldn't respond.  I replaced that by adding `type="tel" pattern="\d+"` to the `input` element instead - this way the browser will enforce that pattern in a friendlier way.  Amelia confirmed that that's desirable.